.changeset/rude-vans-build.md

```diff
@@ -0,0 +1,8 @@
+---
+"@pnpm/outdated": major
+"@pnpm/plugin-commands-installation": major
+"@pnpm/plugin-commands-outdated": major
+"@pnpm/types": major
```

Comment:
it is not a major change but a minor one. Major changes are breaking changes.

Reply:
OK, I will fix this. I thought wrongly that it is breaking change, but you are right. Thanks :+1:

Reply:
fixed in [ea8b3ec](https://github.com/pnpm/pnpm/pull/5408/commits/ea8b3ecabebe3c3c39e851f897e765378dd331f9). Thanks!